letsencrypt/storage.py

```diff
@@ -251,6 +267,7 @@ def current_target(self, kind):
             raise errors.CertStorageError("unknown kind of item")
         link = getattr(self, kind)
         if not os.path.exists(link):
+            logger.debug("Target %s of kind %s does not exist.", link, kind)
```

Comment:
Should be something like `logger.debug("Expected symlink %s does not exist", link)`.